### PR TITLE
Uninstall will wait untill all PVs using OCS storageClasses are removed

### PR DESCRIPTION
**Pull request by bindrad**
Signed-off-by: bindrad <dbindra@redhat.com>

### CODE REVIEW

File: controllers/managedocs_controller_test.go [hunk 1]
```diff
@@ -192,38 +192,49 @@ var _ = Describe("ManagedOCS controller", func() {
...
-					corev1.ResourceStorage: resource.MustParse("1Gi"),
+			Capacity: corev1.ResourceList{
+				corev1.ResourceStorage: resource.MustParse("1Gi"),
+			},
+			PersistentVolumeSource: corev1.PersistentVolumeSource{
+				RBD: &corev1.RBDPersistentVolumeSource{
+					CephMonitors: []string{
+						"0.0.0.0:6789",
+					},
+					RBDImage: "test",
```
Comment (Madhu-1) [inline controllers/managedocs_controller_test.go:214]: the source should be CSI, not RBD nor Cephfs.

Comment (bindrad) [reply controllers/managedocs_controller_test.go:214]: But this is just for our unit tests, we just need to make sure that the PVs are using OCS storage classes

Comment (nb-ohad) [reply controllers/managedocs_controller_test.go:214]: I agree with Dhruv on this one, our unit tests are mocking the stoagecluster and as such we have no notion of CSI.

**Review by nb-ohad — CHANGES_REQUESTED** [review]

File: controllers/managedocs_controller_test.go (hunk 1 above)
Comment (Madhu-1) [reply controllers/managedocs_controller_test.go:214]: >But this is just for our unit tests, we just need to make sure that the PVs are using OCS storage classes

there is no relationship between the OCS storageclasses and the PersistentVolumeSource which is RBD and Cephfs mentioned here.

Comment (bindrad) [reply controllers/managedocs_controller_test.go:214]: Yes, I agree but we just want PVs using OCS storageClasses
It doesn't matter what PersistentVolumeSource we use

**Pull request merged**